### ISSUE TITLE
Allow setting default subtitle language

### ISSUE THREAD

**Issue by flagg19** [Xonshiz/anime-dl#29]
In reference to https://github.com/Xonshiz/anime-dl/issues/26

> ...the default subtitle language is hard-coded as English, can it be turned into a new script parameter?

**Comment by leptoid** [comment]
Can't the preferred subtitle language just be set in your player, such as VLC, though I do see the inconvenience if English isn't your primary language.
I set English as my default subtitle language in all my video players to avoid this issue.

**Comment by Xonshiz** [comment]
Yes, it can be done. But, it also depends on whether the language of the subtitle has been set or not. With anime-dl, most (if not all) subs have the language set. This feature requests wants to set the default language, so as to play default subs on the other players as well. More like a universal setting.

**Comment by IInvocation** [comment]
This would also be cool for plex users. As a german - using this - i mostly set the Library Language to english in order to enjoy conistent show-titles fetched from thetvdb.com - however - now even including an english subtitle will always yield an english sub although there's a german sub available. So - excluding all but a certain language would be pretty cool.
